api/sixtyfps-rs/lib.rs

```diff
@@ -247,9 +263,40 @@ pub fn run_event_loop() {
         .run_event_loop(sixtyfps_corelib::backend::EventLoopQuitBehavior::QuitOnLastWindowClosed);
 }
 
-/// This trait describes the common public API of a strongly referenced SixtyFPS component,
-/// held by a [vtable::VRc]. It allows creating strongly-referenced clones, a conversion into
-/// a weak pointer as well as other convenience functions.
+/// Adds the specified boxed function to an internal queue, notifies the event loop to wake up.
```

Comment:
I guess the function is not boxed anymore - or that's an implementation detail :)